src/main/webapp/src/constants/Navbar.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {GoogleLogin, GoogleLogout} from 'react-google-login'
+
+import 'bulma/css/bulma.css';
+
+/* Component for web app navigation bar */
+const Navbar = () => {
+  // Create state for login/logout status
+  const [isLoggedIn, updateIsLoggedIn] = React.useState(false);
+
+  // Initialize google auth api information
+  const clientID = "";
+  const discoveryDocs = "https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest";
+  const scope = "https://www.googleapis.com/auth/calendar.readonly";
+
+  // Handle successful login
+  const handleLogin = (res) => {
+    updateIsLoggedIn(true);
+  }
+
+  // Handle successful logout
```

Comment:
Lines 59-83 can be made more readable. Just a suggestion, not so important. :)

Reply:
I had to use a ternary operator for a conditional expression if statements aren't allowed. But I'll go ahead and space things out to hopefully make it clearer.

Reply:
I added some more comments too.